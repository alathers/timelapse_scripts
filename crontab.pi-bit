0,10,20,30,40,50 * * * * (/usr/bin/rsync --remove-source-files --progress --stats -rltDv 192.168.1.101:TimeLapse /mnt/AnselAdams/ && echo "last sync at `date`" > /home/alathers/snapsync.log)
55 * * * * /home/alathers/bin/timelapse-housekeeping.sh
