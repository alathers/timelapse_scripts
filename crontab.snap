*/10 * * * * /home/alathers/bin/timelapse.sh /home/alathers/TimeLapse/Paris/AveDItalie/picam
*/2 * * * * /home/alathers/bin/timelapse-usb.sh /home/alathers/TimeLapse/Paris/AveDItalie/usbcam
